src/components/ReviewVotes.jsx

```diff
@@ -0,0 +1,60 @@
+import { patchReviewVotes } from "../routes/api";
+import upvote from '../media/upvote.png'
+import downvote from "../media/downvote.png"
+import upUnselected from "../media/up arrow white.png"
+import downUnselected from "../media/down arrow white.png"
+import { useState } from 'react'
+
+export default function ReviewVotes ({votes, reviewId}) {
+    const [voteCount, setVotes] = useState(votes);
+    const [upClicked, setUpClicked] = useState(false);
+    const [downClicked, setDownClicked] = useState(false); 
+    const [upVoteImg, setUpVoteImg] = useState(upUnselected);
+    const [downVoteImg, setDownVoteImg] = useState(downUnselected);
+
+    function handleUpVote() {
+        if(upClicked){
+            setVotes(votes)
+            setUpClicked(false)
+            setDownClicked(false)
+            setUpVoteImg(upUnselected)
+            setDownVoteImg(downUnselected)
+            patchReviewVotes(reviewId, -1 )
+        }
+        if(!upClicked) {
+            setVotes(votes + 1)
+            setUpClicked(true)
+            setDownClicked(false)
+            setUpVoteImg(upvote)
+            setDownVoteImg(downUnselected)
+            patchReviewVotes(reviewId, 1)
+        }
+    }
+    

```

Comment:
Nice work, though quite a lot of repetition in both the votes components that could be improved if you were ever to refactor